### ISSUE TITLE
some examples are giving compilation error

### ISSUE THREAD

**Issue by felipeweb** [PalmStoneGames/polymer#15]
https://github.com/PalmStoneGames/polymer/blob/master/examples/template-dom-bind/main.go#L33

```
main.go:33:82: cannot use data (variable of type *Data) as code.palmstonegames.com/polymer.BindInterface value in argument to polymer.GetDocument().GetElementByID("tmpl").(*polymer.AutoBindGoTemplate).Bind: wrong type for method Notify
```

Using GopherJS version 

```
GopherJS 1.7-1
```

**Comment by luna-duclos** [comment]
Will check soon

**Comment by luna-duclos** [comment]
Fixed with  43951ec
